docs/_layouts/default.html

```diff
@@ -23,16 +23,25 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Apache Flink: {{ page.title }}</title>
-    <link rel="shortcut icon" href="{{ site.baseurl }}favicon.ico" type="image/x-icon">
-    <link rel="icon" href="{{ site.baseurl }}favicon.ico" type="image/x-icon">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/bootstrap.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/bootstrap-lumen-custom.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/syntax.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/custom.css">
-    <link href="{{ site.baseurl }}css/main/main.css" rel="stylesheet">
+    <link rel="shortcut icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
+    <link rel="icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/bootstrap.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/bootstrap-lumen-custom.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/syntax.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/custom.css">
+    <link href="{{ site.baseurl }}/css/main/main.css" rel="stylesheet">
```

Comment:
Local testing should work out of the box. We could set `{{site.baseurl}}` to http://ci.apache.org/projects/flink/flink-docs-master/ and then change our build_docs script to serve the website for local testing with the `--baseurl http://127.0.01/` parameter. If you want, you can fix this here or we open a JIRA for that.

http://jekyllrb.com/docs/configuration/#serve-command-options